package-support-policy.md

```diff
@@ -0,0 +1,372 @@
+| System.Formats.Asn1                                          | 8.0                  |
+| System.Formats.Cbor                                          | 8.0                  |
+| System.IO.Hashing                                            | 8.0                  |
+| System.IO.Packaging                                          | 8.0                  |
+| System.IO.Pipelines                                          | 8.0                  |
+| System.IO.Ports                                              | 8.0                  |
+| System.Management                                            | 8.0                  |
+| System.Memory.Data                                           | 8.0                  |
+| System.Net.Http.Json                                         | 8.0                  |
+| System.Net.Http.WinHttpHandler                               | 8.0                  |
+| System.Numerics.Tensors                                      | 8.0                  |
+| System.Reflection.Context                                    | 8.0                  |
+| System.Reflection.Metadata                                   | 8.0                  |
+| System.Reflection.MetadataLoadContext                        | 8.0                  |
+| System.Resources.Extensions                                  | 8.0                  |
+| System.Runtime.Caching                                       | 8.0                  |
+| System.Runtime.Serialization.Schema                          | 8.0                  |
+| System.Security.Cryptography.Cose                            | 8.0                  |
+| System.Security.Cryptography.Pkcs                            | 8.0                  |
+| System.Security.Cryptography.ProtectedData                   | 8.0                  |
+| System.Security.Cryptography.Xml                             | 8.0                  |
+| System.Security.Permissions                                  | 8.0                  |
+| System.ServiceModel.Syndication                              | 8.0                  |
+| System.ServiceProcess.ServiceController                      | 8.0                  |
+| System.Speech                                                | 8.0                  |
+| System.Text.Encoding.CodePages                               | 8.0                  |
+| System.Text.Encodings.Web                                    | 8.0                  |
+| System.Text.Json                                             | 8.0                  |
+| System.Threading.AccessControl                               | 8.0                  |
+| System.Threading.Channels                                    | 8.0                  |
+| System.Threading.RateLimiting                                | 8.0                  |
+| System.Threading.Tasks.Dataflow                              | 8.0                  |
+| System.Windows.Extensions                                    | 8.0                  |
```

Comment:
From our conversation @ericstj, I should remove the packages that we build in the latest version.

I will keep those that we stopped building in older versions though (the 3 ones at the end of the table).

```suggestion
```